recipes/gnuplot/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set pkgname = "gnuplot" %}
+{% set version = "5.0.5" %}
+{% set sha256 = "25f3e0bf192e01115c580f278c3725d7a569eb848786e12b455a3fda70312053" %}
+package:
+  name: {{ pkgname }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkgname }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/{{ pkgname }}/files/{{ pkgname }}/{{ version }}/{{ pkgname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - pkgconfig
+    - ncurses 5.9*
+    - readline 6.2*
+    - libgd
+    - cairo 1.14.*
+    - qt 4.8.*
+    - pango >1.10
+  run:
+    - ncurses 5.9*
+    - readline 6.2*
+    - libgd
+    - cairo 1.14.*
+    - qt 4.8.*
+    - pango >1.10
```

Comment:
I didn't find pango in [pin_the_slow_way.py](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py), so I decided to use the minimum requirement.
Do we want to put it into the pinning file? Otherwise it will not get updated to the latest versions.